paper-dropdown-menu-light.html

```diff
@@ -110,55 +110,55 @@
       /**
        * All of these styles below are for styling the fake-input display
        */
-      .dropdown-trigger {
+      [slot="dropdown-trigger"] {
         box-sizing: border-box;
         position: relative;
         width: 100%;
         padding: 16px 0 8px 0;
       }
 
-      :host([disabled]) .dropdown-trigger {
+      :host([disabled]) [slot="dropdown-trigger"] {
         pointer-events: none;
         opacity: var(--paper-dropdown-menu-disabled-opacity, 0.33);
       }
 
-      :host([no-label-float]) .dropdown-trigger {
+      :host([no-label-float]) [slot="dropdown-trigger"] {
         padding-top: 8px;   /* If there's no label, we need less space up top. */
       }
 
       #input {
-        @apply(--paper-font-subhead);
-        @apply(--paper-font-common-nowrap);
+        @apply --paper-font-subhead;
+        @apply --paper-font-common-nowrap;
         line-height: 1.5;
-        border-bottom: 1px solid var(--paper-dropdown-menu-color, --secondary-text-color);
-        color: var(--paper-dropdown-menu-color, --primary-text-color);
+        border-bottom: 1px solid var(--paper-dropdown-menu-color, var(--secondary-text-color));
+        color: var(--paper-dropdown-menu-color, var(--primary-text-color));
         width: 100%;
         box-sizing: border-box;
         padding: 12px 20px 0 0;   /* Right padding so that text doesn't overlap the icon */
         outline: none;
-        @apply(--paper-dropdown-menu-input);
+        @apply --paper-dropdown-menu-input;
       }
 
-      :host-context([dir="rtl"]) #input {
+      :host(:dir(rtl)) #input {
```

Comment:
I think we should keep `host-context` because a) this element needs to work in 1.0 where `:dir` doesn't exist and b) `:dir` isn't actually implemented anywhere that isn't ff